DXRModules/DeusEx/Classes/DXRStats.uc

```diff
@@ -43,22 +43,66 @@ function IncMissionTimer(int mission)
     local name flag;
     local int time;
 
+    local DataStorage datastorage;
+
+
     if (mission < 1) {
         return;
     }
 
+    datastorage = class'DataStorage'.static.GetObj(dxr);
+
+    //Track both the "success path" time (via flags) and
+    //the complete time (via datastorage)
     if (InGame()) {
+        //Success Path
         flagname = "DXRando_Mission"$mission$"_Timer";
         flag = StringToName(flagname);
         time = dxr.flagbase.GetInt(flag);
         dxr.flagbase.SetInt(flag,time+1,,999);
+
+        //Complete Time
+        flagname = "DXRando_Mission"$mission$"_Complete_Timer";
+        flag = StringToName(flagname);
+
+        time = int(datastorage.GetConfigKey(flag));
+        datastorage.SetConfig(flag, time+1, 3600*24*366);
+
     } else {
+        //Success Path
```

Comment:
2 different success paths?

Reply:
"success path", as in "the playthrough that lead to your success", as opposed to a code-based success path